ui/assets/css/_fonts.scss

```diff
@@ -0,0 +1,52 @@
+/*
+ Fonts
+ ==========================================================================
+ Place all @font-face rules in fonts.scss
+ */
+
+ @font-face {
+   font-family: 'Source Sans Pro';
+   font-style: normal;
+   font-weight: 300;
+   src: url('/font/SourceSansPro-Light.ttf');
```

Comment:
Should these be relative, like `../font/[name].ttf`? I'm seeing 404s at this absolute URL.